specification/batch/data-plane/Microsoft.Batch/stable/2021-06-01.14.0/BatchService.json

```diff
@@ -14119,7 +14119,18 @@
           ],
           "x-ms-enum": {
             "name": "StatusLevelTypes",
-            "modelAsString": false
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "error"
```

Comment:
There's a case mismatch between these values and the ones in the "enum" property. I *think* we determined that the UpperCamelCase values were correct.

Reply:
Thanks. I put it back but forgot to adjust the case.